### ISSUE TITLE
Every loot is throwing SetTexture errors with latest update

### ISSUE THREAD

**Issue by Xiuhtec** [Mctalian/RPGLootFeed#128]
Basically everything I pick up throws an error tonight, just updated to v1.7.0.

Some sample errors (I have dozens from just playing for a few minutes):
```
2x ...y/LootDisplayFrame/LootDisplayRow/LootDisplayRow.lua:266: attempt to call method 'SetTexture' (a nil value)
[string "@RPGLootFeed/LootDisplay/LootDisplayFrame/LootDisplayRow/LootDisplayRow.lua"]:266: in function `UpdateIcon'
[string "@RPGLootFeed/LootDisplay/LootDisplay.lua"]:120: in function <...rfaceRPGLootFeed/LootDisplay/LootDisplay.lua:80>
[string "@RPGLootFeed/LootDisplay/LootDisplay.lua"]:147: in function <...rfaceRPGLootFeed/LootDisplay/LootDisplay.lua:137>
[string "=[C]"]: ?
[string "@ElkBuffBars/libs/AceBucket-3.0-4/AceBucket-3.0.lua"]:69: in function <...Ons/ElkBuffBars/libs/AceBucket-3.0/AceBucket-3.0.lua:67>
[string "@ElkBuffBars/libs/AceBucket-3.0-4/AceBucket-3.0.lua"]:85: in function `func'
[string "@AnnounceRare/Libraries/Ace3/AceTimer-3.0-17/AceTimer-3.0.lua"]:57: in function <...nceRare/Libraries/Ace3/AceTimer-3.0/AceTimer-3.0.lua:50>

Locals:
self = Frame {
 RightBorder = LootDisplayFrameRightBorder {
 }
 amount = 2
 Background = Texture {
 }
 Icon = Button {
 }
 TopBorder = LootDisplayFrameTopBorder {
 }
 icon = 1387364
 link = "|cffffffff|Hitem:220137::::::::80:63:::::::::|h[Bismuth Bitterling]|h|r"
 BottomBorder = LootDisplayFrameBottomBorder {
 }
 key = "220137"
 _prev = Frame {
 }
 HighlightAnimation = <table> {
 }
 AmountText = FontString {
 }
 LeftBorder = LootDisplayFrameLeftBorder {
 }
 FadeOutAnimation = AnimationGroup {
 }
}
key = "220137"
icon = 1387364
quality = nil
(*temporary) = nil
(*temporary) = Button {
 IconOverlay = Texture {
 }
 icon = LootDisplayFrameIconTexture {
 }
 itemContextMatchResult = 3
 searchOverlay = LootDisplayFrameSearchOverlay {
 }
 PushedTexture = Texture {
 }
 NormalTexture = LootDisplayFrameNormalTexture {
 }
 IconOverlay2 = Texture {
 }
 IconBorder = Texture {
 }
 HighlightTexture = Texture {
 }
 Stock = LootDisplayFrameStock {
 }
 ItemContextOverlay = Texture {
 }
 showMatchHighlight = true
 Count = LootDisplayFrameCount {
 }
}
(*temporary) = 1387364
(*temporary) = "attempt to call method 'SetTexture' (a nil value)"
Masque = nil
iconGroup = nil
```
---
```
1x ...y/LootDisplayFrame/LootDisplayRow/LootDisplayRow.lua:266: attempt to call method 'SetTexture' (a nil value)

==== Addon Info RPGLootFeed v1.7.0-1-c1e65b7 ====

Log traces related to Currency
-------------------------------------------------
[00:59:19]{I}(WOW)[CURR]: CURRENCY_DISPLAY_UPDATE (tot: 5) [2815]
[00:59:19]{I}(WOW)[CURR]: CURRENCY_DISPLAY_UPDATE (tot: 1) [3008]
[00:57:19]{I}(RLF)[CURR]: |Hcurrency:3008:0[Valorstones] x15 (tot: 15 (diff: +1)) ~UPDATE~ [3008]
[00:57:19]{I}(WOW)[CURR]: CURRENCY_DISPLAY_UPDATE (tot: 1) [3008]
[00:57:19]{I}(RLF)[CURR]: |Hcurrency:2815:0[Resonance Crystals] x41 (tot: 41 (diff: +3)) ~UPDATE~ [2815]
[00:57:19]{I}(WOW)[CURR]: CURRENCY_DISPLAY_UPDATE (tot: 3) [2815]
[00:57:19]{I}(RLF)[CURR]: |Hcurrency:3008:0[Valorstones] x14 (tot: 14 (diff: +1)) ~UPDATE~ [3008]
[00:57:19]{I}(WOW)[CURR]: CURRENCY_DISPLAY_UPDATE (tot: 1) [3008]
[00:57:19]{I}(RLF)[CURR]: |Hcurrency:2815:0[Resonance Crystals] x38 (tot: 38 (diff: +3)) ~UPDATE~ [2815]
[00:57:19]{I}(WOW)[CURR]: CURRENCY_DISPLAY_UPDATE (tot: 3) [2815]
-------------------------------------------------

Please report this issue @ github: McTalian/RPGLootFeed


[string "@RPGLootFeed/LootDisplay/LootDisplayFrame/LootDisplayRow/LootDisplayRow.lua"]:266: in function `UpdateIcon'
[string "@RPGLootFeed/LootDisplay/LootDisplay.lua"]:120: in function <...rfaceRPGLootFeed/LootDisplay/LootDisplay.lua:80>
[string "@RPGLootFeed/LootDisplay/LootDisplay.lua"]:76: in function `ShowLoot'
[string "@RPGLootFeed/Features/LootDisplayProperties.lua"]:34: in function `Show'
[string "@RPGLootFeed/Features/Currency.lua"]:103: in function <RPGLootFeed/Features/Currency.lua:96>
[string "=[C]"]: ?
[string "@RPGLootFeed/Core.lua"]:48: in function `fn'
[string "@RPGLootFeed/Features/Currency.lua"]:96: in function `?'
[string "@AdvancedInterfaceOptions/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:109: in function <...ons/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@AdvancedInterfaceOptions/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:19: in function <...ons/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@AdvancedInterfaceOptions/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@TomTom/libs/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <...aceTomTom/libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
self = Frame {
 RightBorder = LootDisplayFrameRightBorder {
 }
 amount = 5
 Background = Texture {
 }
 Icon = Button {
 }
 _prev = Frame {
 }
 TopBorder = LootDisplayFrameTopBorder {
 }
 link = "|cffffffff|Hcurrency:2815:0|h[Resonance Crystals]|h|r"
 BottomBorder = LootDisplayFrameBottomBorder {
 }
 key = 2815
 icon = 2967113
 HighlightAnimation = <table> {
 }
 AmountText = FontString {
 }
 LeftBorder = LootDisplayFrameLeftBorder {
 }
 FadeOutAnimation = AnimationGroup {
 }
}
key = 2815
icon = 2967113
quality = 1
(*temporary) = nil
(*temporary) = Button {
 IconOverlay = Texture {
 }
 icon = LootDisplayFrameIconTexture {
 }
 itemContextMatchResult = 3
 searchOverlay = LootDisplayFrameSearchOverlay {
 }
 PushedTexture = Texture {
 }
 NormalTexture = LootDisplayFrameNormalTexture {
 }
 IconOverlay2 = Texture {
 }
 IconBorder = Texture {
 }
 HighlightTexture = Texture {
 }
 Stock = LootDisplayFrameStock {
 }
 ItemContextOverlay = Texture {
 }
 showMatchHighlight = true
 Count = LootDisplayFrameCount {
 }
}
(*temporary) = 2967113
(*temporary) = "attempt to call method 'SetTexture' (a nil value)"
Masque = nil
iconGroup = nil

```
---

```
1x ...y/LootDisplayFrame/LootDisplayRow/LootDisplayRow.lua:266: attempt to call method 'SetTexture' (a nil value)

==== Addon Info RPGLootFeed v1.7.0-1-c1e65b7 ====

Log traces related to ItemLoot
-------------------------------------------------
[00:58:39]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive loot: |Hitem:220295::::::::80:63:::::::::[Unidentified Viscous Substance]
[00:57:20]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive item: |Hitem:220137::::::::80:63:::::::::[Bismuth Bitterling]x2
[00:57:20]{I}(RLF)[ITEM]: |Hitem:219150::::::::80:63:::::::::[Pile of Rusted Scrap] x5 (tot: 5 (diff: +1)) ~UPDATE~ [219150]
[00:57:20]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive item: |Hitem:219150::::::::80:63:::::::::[Pile of Rusted Scrap]
[00:57:20]{I}(RLF)[ITEM]: |Hitem:212493::::::::80:63:::::::::[Odd Glob of Wax] x3 (tot: 3 (diff: +1)) ~UPDATE~ [212493]
[00:57:20]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive item: |Hitem:212493::::::::80:63:::::::::[Odd Glob of Wax]
[00:57:19]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive item: |Hitem:225767::::::::80:63::::1:28:81:::::[Spare Toolbox]
[00:57:19]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive item: |Hitem:210796::::::::80:63::::1:38:1:::::[Mycobloom |A:Professions-ChatIcon-Quality-Tier1:17:15::1|a]x2
[00:57:19]{I}(WOW)[ITEM]: CHAT_MSG_LOOT You receive item: |Hitem:212242::::::::80:63::::1:38:1:::::[Cavedweller's Delight |A:Professions-ChatIcon-Quality-Tier1:17:15::1|a]
[00:57:19]{I}(RLF)[ITEM]: |Hitem:219150::::::::80:63:::::::::[Pile of Rusted Scrap] x4 (tot: 4 (diff: +1)) ~UPDATE~ [219150]
-------------------------------------------------

Please report this issue @ github: McTalian/RPGLootFeed


[string "@RPGLootFeed/LootDisplay/LootDisplayFrame/LootDisplayRow/LootDisplayRow.lua"]:266: in function `UpdateIcon'
[string "@RPGLootFeed/LootDisplay/LootDisplay.lua"]:120: in function <...rfaceRPGLootFeed/LootDisplay/LootDisplay.lua:80>
[string "@RPGLootFeed/LootDisplay/LootDisplay.lua"]:76: in function `ShowLoot'
[string "@RPGLootFeed/Features/LootDisplayProperties.lua"]:34: in function `Show'
[string "@RPGLootFeed/Features/ItemLoot.lua"]:118: in function <RPGLootFeed/Features/ItemLoot.lua:110>
[string "=[C]"]: ?
[string "@RPGLootFeed/Core.lua"]:48: in function `fn'
[string "@RPGLootFeed/Features/ItemLoot.lua"]:139: in function `?'
[string "@AdvancedInterfaceOptions/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:109: in function <...ons/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:109>
[string "=[C]"]: ?
[string "@AdvancedInterfaceOptions/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:19: in function <...ons/libs/CallbackHandler-1.0/CallbackHandler-1.0.lua:15>
[string "@AdvancedInterfaceOptions/libs/CallbackHandler-1.0-8/CallbackHandler-1.0.lua"]:54: in function `Fire'
[string "@TomTom/libs/AceEvent-3.0-4/AceEvent-3.0.lua"]:120: in function <...aceTomTom/libs/AceEvent-3.0/AceEvent-3.0.lua:119>

Locals:
self = Frame {
 RightBorder = LootDisplayFrameRightBorder {
 }
 amount = 1
 Background = Texture {
 }
 Icon = Button {
 }
 TopBorder = LootDisplayFrameTopBorder {
 }
 icon = 1391773
 link = "|cff9d9d9d|Hitem:220295::::::::80:63:::::::::|h[Unidentified Viscous Substance]|h|r"
 BottomBorder = LootDisplayFrameBottomBorder {
 }
 key = "220295"
 _prev = Frame {
 }
 HighlightAnimation = <table> {
 }
 AmountText = FontString {
 }
 LeftBorder = LootDisplayFrameLeftBorder {
 }
 FadeOutAnimation = AnimationGroup {
 }
}
key = "220295"
icon = 1391773
quality = nil
(*temporary) = nil
(*temporary) = Button {
 IconOverlay = Texture {
 }
 icon = LootDisplayFrameIconTexture {
 }
 itemContextMatchResult = 3
 searchOverlay = LootDisplayFrameSearchOverlay {
 }
 PushedTexture = Texture {
 }
 NormalTexture = LootDisplayFrameNormalTexture {
 }
 IconOverlay2 = Texture {
 }
 IconBorder = Texture {
 }
 HighlightTexture = Texture {
 }
 Stock = LootDisplayFrameStock {
 }
 ItemContextOverlay = Texture {
 }
 showMatchHighlight = true
 Count = LootDisplayFrameCount {
 }
}
(*temporary) = 1391773
(*temporary) = "attempt to call method 'SetTexture' (a nil value)"
Masque = nil
iconGroup = nil

```

**Comment by Mctalian** [comment]
Thanks for the report! I'll get this fixed right away

**Comment by Mctalian** [comment]
Same as #127 going to close this one out

**Comment by Mctalian** [comment]
Thanks for your patience, should be fixed in v1.7.1!
